### PR TITLE
Make sure only article pages will be promoted with content rotation

### PR DESCRIPTION
**Pull request by saeedmb8**

### CODE REVIEW

File: molo/core/management/commands/remove_all_featured_articles.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    def handle(self, **options):
+        article_pages = ArticlePage.objects.all()
+        for article in article_pages:
+            if article.featured_in_latest or \
+                article.featured_in_latest_start_date or \
+                    article.featured_in_latest_end_date:
+                article.featured_in_latest = False
+                article.featured_in_latest_start_date = None
+                article.featured_in_latest_end_date = None
+                article.save_revision().publish()
```
Comment (miltontony) [inline molo/core/management/commands/remove_all_featured_articles.py]: This should just be `save`

File: molo/core/management/commands/remove_all_featured_articles.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+from __future__ import absolute_import, unicode_literals
+
+from django.core.management.base import BaseCommand
+from molo.core.models import ArticlePage
+
+
+class Command(BaseCommand):
+    def handle(self, **options):
+        article_pages = ArticlePage.objects.all()
```
Comment (miltontony) [inline molo/core/management/commands/remove_all_featured_articles.py]: you can do it like this
```python
ArticlePage.objects.all()update(
    featured_in_latest=False, 
    featured_in_latest_start_date=None, 
    featured_in_latest_end_date=None)
```

File: molo/core/tasks.py [hunk 3]
```diff
@@ -115,9 +115,12 @@ def rotate_latest(main_lang, index, main, site_settings, day):
     def demote_last_featured_article():
         # set the last featured_in_latest article to false
         article = main.latest_articles().live().last()
```
Comment (miltontony) [inline molo/core/tasks.py]: remove this

**Comment by codiebeulaine** [comment]
👍

**Pull request merged**